Data/models/Bookmark.swift

```diff
@@ -238,12 +252,10 @@ public final class Bookmark: NSManagedObject, WebsitePresentable, Syncable, CRUD
     }
     
     public class func add(from list: [(url: URL, title: String)]) {
-        let context = DataController.newBackgroundContext()
-        context.performAndWait {
+        DataController.performBackgroundTask { context in
```

Comment:
This is no longer synchronous, does this impact anything?

Reply:
Not from my observations, it adds the preloaded favorites which are automatically fetched by fetch results controller.